src/main/java/com/byndyusoft/java/spring/microservice/template/config/package-info.java

```diff
@@ -0,0 +1,6 @@
+/**
+ * Place for bean configuration {@link org.springframework.context.annotation.Configuration}.
+ *
+ */
+
+package com.byndyusoft.java.spring.microservice.template.config;
```

Comment:
config package already have class from this package, why this stub need? (also `exception`, `model`, `repository`, `service`, `controller`, `handler` packages)

Reply:
Discussed